### PR TITLE
fix: added missing library in package

### PR DESCRIPTION
**Pull request by Mylothros-Charalampos**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "react-leaflet-google-v2",
+  "name": "react-leaflet-google-v3",
```
Comment (Charmatzis) [inline package.json]: Why v3?

File: example/google.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import { MapContainer, TileLayer, LayersControl } from 'react-leaflet'
-import { GoogleLayer } from '../src'
+//import { GoogleLayer } from '../src'
```
Comment (Charmatzis) [inline example/google.js:3]: please remove that

**Review by Charmatzis — CHANGES_REQUESTED** [review]

File: example/google.js (hunk 2 above)
Comment (Mylothros-Charalampos) [reply example/google.js:3]: it's there so you can see the error when we run this by importing GoogleLayer from dist when we import it from src there is no error

File: package.json (hunk 1 above)
Comment (Mylothros-Charalampos) [reply package.json]: my fault I am going to fix it

**Review by Charmatzis — APPROVED** [review]

**Pull request merged**